src/main/java/pages/BasePage.java

```diff
@@ -66,6 +66,7 @@ protected void typeText(WebElement element, String text) {
         waitForElementVisibility(element);
         element.clear();
         element.sendKeys(text);
+        element.sendKeys(Keys.ENTER);
```

Comment:
This line press down Enter-key every time the text field is filled. AddOwner button in AddOwnerPageTest can't be located and pressed because of this Enter and all the tests using that button are failed